src/app/app.component.ts

```diff
@@ -1,15 +1,27 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { Broadcaster } from './shared/broadcaster.service';
 import { AuthenticationService } from './auth/authentication.service';
 
 @Component({
   selector: 'alm-app',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
-  })
-export class AppComponent {
+})
+export class AppComponent implements OnInit {
   
-  constructor(auth: AuthenticationService){
+  constructor(private auth: AuthenticationService, private broadcaster: Broadcaster){
     //auth.isLoggedIn();
   }
-}
\ No newline at end of file
+  loggedIn: Boolean = false;
```

Comment:
[class.containerPadd-nologin] = "!loggedIn" I am toggling the state to know whether user is logged in or not. If not then removing the padding -bottom